packages/web/config/chain-infos.ts

```diff
@@ -468,7 +468,7 @@ const chainInfos = (
           coinMinimalDenom:
             "cw20:juno1gz8cf86zr4vw9cjcyyv432vgdaecvr9n254d3uwwkx9rermekddsxzageh:GKEY",
           coinDecimals: 6,
-          // coinGeckoId: "pool:gkey",
+          coinGeckoId: "pool:gkey",
```

Comment:
this id is not defined (but you can do so in /config/price.ts). No point in uncommenting this without it defined.